pkg/controller/cloneset/utils/cloneset_utils.go

```diff
@@ -0,0 +1,61 @@
+package utils
+
+import (
+	"context"
+
+	appsv1alpha1 "github.com/openkruise/kruise/pkg/apis/apps/v1alpha1"
+	apps "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	podutil "k8s.io/kubernetes/pkg/api/v1/pod"
+	kubecontroller "k8s.io/kubernetes/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var ControllerKind = appsv1alpha1.SchemeGroupVersion.WithKind("CloneSet")
+
+func GetActivePods(reader client.Reader, namespace string) ([]*v1.Pod, error) {
```

Comment:
exported functions or variables should be commented

Reply:
done